tests/helpers.py

```diff
@@ -0,0 +1,20 @@
+import asyncio
+from unittest import mock
+
+
+def make_coro_mock():
```

Comment:
The right way to do this is creating a "fixture" because the tests would receive this coroutine as an explicit parameter.